capnp/src/message.rs

```diff
@@ -708,10 +708,10 @@ fn test_allocate_max() {
     allocator.deallocate_segment(a3, s3, 0);
 }
 
-impl Builder<HeapAllocator> {
+impl Default for Builder<HeapAllocator> {
     /// Constructs a new `message::Builder<HeapAllocator>` whose first segment has length
     /// `SUGGESTED_FIRST_SEGMENT_WORDS`.
-    pub fn new_default() -> Builder<HeapAllocator> {
```

Comment:
Abruptly deleting this method would cause unnecessary pain. It would be better to keep it around and deprecate it with a warning like "use the default() method instead (from the Default trait)".

Reply:
in what version should each of these be deprecated?

Reply:
ping @dwrensha

Reply:
i think they meant the deprecated hint: https://doc.rust-lang.org/reference/attributes/diagnostics.html#the-deprecated-attribute

Reply:
> i think they meant the deprecated hint: https://doc.rust-lang.org/reference/attributes/diagnostics.html#the-deprecated-attribute

exactly. This attribute has the `since` argument, which defines the version in which it should be deprecated. Hence my question.

Reply:
wouldn't that just be the current version upon release?